docs/pages/using-nhs-notify/personalisation.md

```diff
@@ -18,47 +18,37 @@ To personalise a message, use double brackets to add a placeholder to your conte
     text='<code>Hello ((firstName)), your NHS Number is ((nhsNumber))'
```

Comment:
Missing a closing </code> tag.

Should be:
`text='<code>Hello ((firstName)), your NHS Number is ((nhsNumber))</code>'`

Reply:
Not sure this is supposed to have <code> tags at all, will remove for the time being.